client/src/components/atoms/Button.module.scss

```diff
@@ -13,10 +13,10 @@
     font-weight: $font-weight-bold;
     text-transform: uppercase;
     border-radius: 2px;
-    transition: .2s ease-out;
+    transition: 0.2s ease-out;
     color: $brand-white;
     background: $brand-grey-light;
-    box-shadow: 0 9px 18px 0 rgba(0, 0, 0, .1);
+    box-shadow: 0 9px 18px 0 rgba(0, 0, 0, 0.1);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/oceanprotocol/commons/pullRequest?prid=5548133)